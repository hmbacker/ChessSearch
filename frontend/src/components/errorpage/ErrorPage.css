.error {
  background-color: rgb(31, 32, 36);
  height: 100vh;
  align-content: center;
  display: table;
}
.error-p {
  color: white;
  text-align: center;
  line-height: 100px;
  vertical-align: middle;
  display: table-cell;
  position: absolute;
  top: 40%;
  left: 45%;
}
