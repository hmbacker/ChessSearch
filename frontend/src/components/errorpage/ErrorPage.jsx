import React from "react";
import "./ErrorPage.css";
import knight from "../../media/search.svg";
import { Link } from "react-router-dom";

// 404 page
function GamePageError() {
  return (
    <div className="error">
      <Link className="search" to="/">
        <img src={knight} alt="white chess knight" />
      </Link>
      <p className="error-p">Page not found</p>
    </div>
  );
}
export default GamePageError;
