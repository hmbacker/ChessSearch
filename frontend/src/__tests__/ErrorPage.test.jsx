import React from "react";
import ErrorPage from "../components/errorpage/ErrorPage";
import { shallow } from "enzyme";

test("renders correctly", () => {
  const wrapper = shallow(<ErrorPage />);
  expect(wrapper.debug()).toMatchSnapshot();
});
