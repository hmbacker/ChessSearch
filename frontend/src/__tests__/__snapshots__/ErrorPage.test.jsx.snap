// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"<div className=\\"error\\">
  <Link className=\\"search\\" to=\\"/\\">
    <img src={{...}} alt=\\"white chess knight\\" />
  </Link>
  <p className=\\"error-p\\">
    Page not found
  </p>
</div>"
`;
